optaweb-vehicle-routing-frontend/pom.xml

```diff
@@ -26,25 +26,46 @@
   </parent>
 
   <artifactId>optaweb-vehicle-routing-frontend</artifactId>
-  <packaging>pom</packaging>
+  <packaging>war</packaging>
 
   <name>OptaWeb Vehicle Routing Frontend</name>
 
   <properties>
     <sonar.sources>src</sonar.sources>
-    <sonar.test.inclusions>optaweb-vehicle-routing-frontend/**/*test.ts,optaweb-vehicle-routing-frontend/**/*test.tsx</sonar.test.inclusions>
+    <sonar.test.inclusions>
+      optaweb-vehicle-routing-frontend/**/*test.ts,optaweb-vehicle-routing-frontend/**/*test.tsx
+    </sonar.test.inclusions>
     <sonar.typescript.lcov.reportPaths>coverage/lcov.info</sonar.typescript.lcov.reportPaths>
+
+    <!-- Frontend properties -->
+    <frontend-maven-plugin.version>1.7.5</frontend-maven-plugin.version>
+    <maven-resources-plugin.version>3.1.0</maven-resources-plugin.version>
   </properties>
 
+
   <build>
+    <finalName>optaweb-vehicle-routing-frontend</finalName>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>com.github.eirslett</groupId>
+          <artifactId>frontend-maven-plugin</artifactId>
+          <version>${frontend-maven-plugin.version}</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-resources-plugin</artifactId>
+          <version>${maven-resources-plugin.version}</version>
```

Comment:
Not needed. The version is already managed in jboss-parent: https://github.com/jboss/jboss-parent-pom/blob/jboss-parent-31/base/pom.xml#L104.